react/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "redux-loop/react",
```

Comment:
on second thought, i kind of thing this package should be `redux-loop/react-redux` since that's the types we're overriding. And the folder should be react-redux as well.

Reply:
I was going to suggest the same.

Reply:
yeah, that makes sense